sound/soc/sof/imx/Makefile

```diff
@@ -2,5 +2,8 @@
 snd-sof-imx8-objs := imx8.o
 snd-sof-imx8m-objs := imx8m.o
 
+snd-sof-imx8-objs += common.o
+snd-sof-imx8m-objs += common.o
+
```

Comment:
This is not OK. This will try to link common.o twice into Image file. If common.c contains non-static symbols then there will be a compilation error.

I would suggest on adding

```snd-sof-imx-common-objs := common.o```